packages/x-date-pickers/src/PickersTextField/PickersTextField.tsx

```diff
@@ -161,9 +161,11 @@ const PickersTextField = React.forwardRef(function PickersTextField(
         ownerState={ownerState}
         {...other}
       >
-        <InputLabel htmlFor={id} id={inputLabelId} {...InputLabelProps}>
-          {label}
-        </InputLabel>
+        {label != null && label !== '' && (
```

Comment:
Aligning with `TextField` on `@mui/material`: https://github.com/mui/material-ui/blob/c3f36426ccbb34368b084fd4b3c5ac65cee43b3e/packages/mui-material/src/TextField/TextField.js#L243
@noraleonte was there a reason to always render a label element in Picker's case?

Reply:
Not entirely sure 🤔 But only rendering when there is a label prop makes more sense 👍 🙈